### ISSUE TITLE
Previews in Info Panel of Office Documents Don't Work

### ISSUE THREAD

**Issue by nathan2423** [flathub/org.kde.dolphin#33]
Hello and thanks for your work on this flatpack. I notice that the Info Panel on the right does not show previews of Microsoft Word and Excel files -- it shows just a dummy image.  Is there a way to get this feature functioning?  - THANK YOU

**Comment by tsdgeos** [comment]
I don't actually have any Word or Excel file around to try, could you please attach one of each?

**Comment by freecicero** [comment]
Thank you I would be glad to.  Attached are two very simple samples.   My understanding of getting this preview to work in the normal KDE desktop is that there are apparently some preview formats installed when you install Calligra, so perhaps there are some kind of filter files that can be lifted from there.  Calligra is a very poor office suite, and not anywhere close to Libreoffice or others, but the ability to preview Word and Excel documents, both in the older doc / xls format and the newer docx / xlsx format, is critically important for use of Dolphin in a modern office situation.  If there is anything you can do to get this working it would be great -- thank you!!!!   I know that previewing these documents is possible in the latest distros such as Kde neon, Kubuntu, Manjaro, etc.

again - thank you!




[Sample Spreadsheet and Word Document.zip](https://github.com/flathub/org.kde.dolphin/files/5102366/Sample.Spreadsheet.and.Word.Document.zip)

**Comment by tsdgeos** [comment]
> Calligra is a very poor office suite

I would appreciate if you did not use such language here. You may not like Calligra, keep it to yourself, or for when someone asks your opinion, there's no need to belittle work done by fellow colleagues of the Free Software community.

**Comment by freecicero** [comment]
SORRY!  I had no intention of belittling anyone and I will delete that from the message if it will let me.  KDE and all its components are a treasure for humanity and they deserve great credit for all their work.

Here is a graphic of how the preview works in Dolphin using Opensuse Tumbleweed with the latest KDE

![Screenshot_20200820_140932](https://user-images.githubusercontent.com/21122734/90770271-823eb500-e2bf-11ea-86b8-f17359cff30c.png)

**Comment by tsdgeos** [comment]
Yes, there's quite a few thumbnail creators in calligra, and a few more in other places we don't include either.

Needs evaluating on whether we can include them all or if we can use the magic flatpak plugin system https://lxr.kde.org/search?_filestring=&_string=%3DThumbCreator

**Comment by tsdgeos** [comment]
Tracking/discussin it upstream https://invent.kde.org/teams/flathub/issues/-/issues/5

**Comment by freecicero** [comment]
Thank you very much for checking into this!  I realize that at some point it is impractical and inappropriate to intergrate into a Dolphin flatpak all of the various services with which it might integrate, but the core viewer functionality is really a killer feature and really puts Dolphin at the head of the pack of all linux file managers for office use.
